src/client/src/scenes/RoomScreen/RoomSettingModal/RoomSettingModal.js

```diff
@@ -84,7 +85,10 @@ export const RoomSettingModal = (props) => {
     }
 
     return (
-        <React.Fragment>
+        <React.Fragment onHide={props.onHide} show={props.show}>
+            <Modal.Header closeButton>
```

Comment:
You can use self closing tags if there is no content inside the component. <Modal.Header closeButton/>